### PR TITLE
Build prebuilds on Ubuntu 20.04

### PR DESCRIPTION
**Pull request by segevfiner**
This is the minimum supported Ubuntu for Node.js 20

### CODE REVIEW

**Comment by ObserverOfTime** [comment]
Make it an input instead (with `latest` being the default).

**Comment by segevfiner** [comment]
Hmm... I can make it an input, but I want `ubuntu-20.04` as the default so that prebuild grammars work on the minimum supported version of Node.js and not only from Ubuntu 22.04 that ubuntu-latest happens to point to ATM.

**Review by segevfiner — COMMENTED** [review]
Review fixes

File: .github/workflows/package-npm.yml [hunk 1]
```diff
@@ -19,6 +19,10 @@ on:
         description: The Emscripten version
         default: ${{vars.EMSCRIPTEN_VERSION || '3.1.55'}}
         type: string
+      runs-on-linux:
+        decsription: The type of Linux machine to run the job on.
```
Comment (clason) [inline .github/workflows/package-npm.yml:23]:
```suggestion
        description: The runner image to run the job on.
```

File: .github/workflows/package-npm.yml [hunk 2]
```diff
@@ -19,6 +19,10 @@ on:
         description: The Emscripten version
         default: ${{vars.EMSCRIPTEN_VERSION || '3.1.55'}}
         type: string
+      runs-on-linux:
+        description: The runner image to run the job on.
+        default: ubuntu-20.04
```
Comment (ObserverOfTime) [inline .github/workflows/package-npm.yml:24]:
```suggestion
      ubuntu-version:
        description: The version of the Ubuntu runner image
        default: ${{vars.UBUNTU_VERSION || '20.04'}}
```

**Review by ObserverOfTime — COMMENTED** [review]
Also add the input to the readme.

File: .github/workflows/package-npm.yml (hunk 2 above)
Comment (ObserverOfTime) [reply .github/workflows/package-npm.yml]: You forgot to change the rest.

Comment (ObserverOfTime) [reply .github/workflows/package-npm.yml]: `ubuntu-${{inputs.ubuntu-version}}`

File: .github/workflows/package-npm.yml [hunk 3]
```diff
@@ -61,7 +65,7 @@ jobs:
     runs-on: ${{matrix.os}}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-${{inputs.ubuntu-version}}, windows-latest, macos-latest]
```
Comment (eed3si9n) [inline .github/workflows/package-npm.yml:68]: I'm getting
```
error parsing called workflow
".github/workflows/publish.yml"
-> "tree-sitter/workflows/.github/workflows/package-npm.yml@main" (source branch with sha:d82dd1521ffdb4b4b71377ffe4c4d7c2f3e15d18)
: You have an error in your yaml syntax on line 67
```

Comment (segevfiner) [reply .github/workflows/package-npm.yml:68]: I got reviewed
Reviewed so hard
Reviewed till it broke my code
🎵

Might need double quotes around the ubuntu one in the array...?

Comment (eed3si9n) [reply .github/workflows/package-npm.yml:68]: Here's my PR for this - https://github.com/tree-sitter/workflows/pull/13

**Pull request merged**